### ISSUE TITLE
[Feature]  echats3D地图type: "map3D"如何关闭hover区域高亮

### ISSUE THREAD

**Issue by yixnhh** [apache/echarts#19720]
### What problem does this feature solve?

type: "map3D"无法关闭hover区域高亮，尝试了很多方法，需要关闭hover高亮保留点击事件

### What does the proposed API look like?

type: "map3D"无法关闭hover区域高亮，尝试了很多方法，需要关闭hover高亮保留点击事件

**Comment by helgasoft** [comment]
yes, [map3D.emphasis](https://echarts.apache.org/en/option-gl.html#series-map3D.emphasis) is missing the _disabled_ property which exists in [2D map](https://echarts.apache.org/en/option.html#series-map.emphasis.disabled)
